### ISSUE TITLE
Library isn't loading.

### ISSUE THREAD

**Issue by Aelf000** [Tsume/Maple-Tree#390]
### Issue Checklist
- [ ✓ ] Meet Requirements [Here](https://github.com/Tsume/Maple-Tree#requirements)
- [ ✓] Read FAQ [Here](https://github.com/Tsume/Maple-Tree#faq)

### Version Used
Example: 2.1.1.18

### Expected behavior
Being able to see my games in the Library

### Actual behavior
Nothing is there.

### Steps to reproduce behavior
1. Lauch Mapleseed
2. Selecting cemu.exe and roms folder.
3. Loading Library.

**Comment by StalkyTalky** [comment]
### **PRESUMABLE FIX**

If you were downloading anything during that time, maybe it's because new data gets downloaded on to your computer that MapleSeed is constantly reading but more and more keeps piling up so MapleSeed gets stuck until the download is finished? Another possible cause is that because there is a download in progress MapleSeed can't read it due to the fact that another resource is using it (Browser, torrent client,WiiUSBhelper (Can be used with a workaround [here](https://www.reddit.com/r/CemuPiracy/comments/8gj8z8/workaround_download_and_run_wii_u_usb_helper/)), ext) and MapleSeed is waiting for it to finish the download before it can read it. In simpler terms: Resource A (MapleSeed) needs resource C (Your game) and resource B (Browser,torrent client, WiiUSBhelper) does not need it. A has used the files and is now done. But if B is still using C, A will have to wait for B to finish before it can use it. I hope this helps or at least explained something to you if you stumble into any future problems!

**Comment by StalkyTalky** [comment]
TL;DR: It's probably because you were downloading a game in the exact ROMs directory which MapleSeed had to wait for the game to finish before it could read it, which is why it kept waiting. The solution for me was to wait until the game had downloaded. (I don't own an SD Card to use to flash my games over, which is why I used the term 'download'. I used Wii USB Helper, there is a workaround for it to make it work, but it got my last comment removed so I won't be linking it.)
